src/main/webapp/WEB-INF/view/admin.jsp

```diff
@@ -0,0 +1,30 @@
+<!-- 
+  Admin.jsp
+-->
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Admin</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+  <!-- 
+    TO-DO(Matthew Oakley) I am going to add the stats for the admin page
+    I just have this basic setup to see if I could get the servlets working
+  -->
+  <nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/conversations">Conversations</a>
+    <% if(request.getSession().getAttribute("user") != null) { %>
```

Comment:
Does the user check still work without setting the UserStore in the servlet?

Reply:
Yes.